### PR TITLE
Added duplicate project interface and functionality

### PR DESCRIPTION
**Pull request by alfeay**

### CODE REVIEW

File: appjs/router.js [hunk 1]
```diff
@@ -193,5 +194,52 @@ module.exports = Backbone.Router.extend({
...
+
+      if ( !blueprint ) {
+        blueprint = new models.Blueprint({ id: project.get('blueprint_id') });
+        app.blueprints.add(blueprint);
+        return blueprint.fetch();
+      }
+    }).then(function() {
+      project.blueprint = blueprint;
+
+      new_attributes.blueprint_config = project.attributes.blueprint_config;
```
Comment (ryanmark) [inline appjs/router.js]: need to use `_.clone` here

File: appjs/router.js [hunk 2]
```diff
@@ -193,5 +194,52 @@ module.exports = Backbone.Router.extend({
...
+      new_attributes.blueprint_id = project.attributes.blueprint_id;
+      new_attributes.blueprint_title = project.attributes.blueprint_title;
+      new_attributes.blueprint_version = project.attributes.blueprint_version;
+      new_attributes.data = project.attributes.data;
```
Comment (ryanmark) [inline appjs/router.js]: need to use `_.clone` here

File: appjs/templates/project.ejs [hunk 3]
```diff
@@ -1,6 +1,14 @@
 <div class="row m-page-heading">
   <div class="col-xs-12">
-    <h3><% if ( model.isNew() ) { %>New Project<% } else { %><%=model.get( 'title' ) %><% } %></h3>
+    <h3>
+        <% if ( model.isNew() && (copyProject == false) ) {
+          %>New Project<%
+        } else if ( copyProject ) {
+          %>New <%= model.get( 'title' ) %><%
```
Comment (ryanmark) [inline appjs/templates/project.ejs]: Should maybe be `Duplicate` instead of `New` ?

**Comment by ryanmark** [comment]
Fix those few comments then you're good to merge and deploy to prod

File: appjs/templates/project.ejs (hunk 3 above)
Comment (alfeay) [inline appjs/templates/project.ejs]: It appends the title after changing the title to `Copy of <%= model.get( 'title' ) %>`, so the final output is `New Copy of <old title>`.

**Pull request merged**